Jenkinsfile

```diff
@@ -67,7 +67,10 @@ stage('Prepare') {
 		checkout scm
 
 		// Load the child Jenkinsfiles.
-		scriptForApps = load('apps/build.groovy')
+		if (params.deploy_env == 'healthapt') {
```

Comment:
https://github.com/CMSgov/beneficiary-fhir-data/blob/18e5c39cc55d62ccfef42fc69f61625193a69d4e/Jenkinsfile#L70-L78

Could this be simplified to:
```
		if (params.deploy_env == 'healthapt') {
			scriptForApps = load('apps/build.groovy')
			scriptForDeploys = load('ops/deploy-healthapt.groovy')
		} else if (params.deploy_env == 'ccs') {
			scriptForApps = load('apps/build-ccs.groovy')
			scriptForDeploys = load('ops/deploy-ccs.groovy')
		}
```

Reply:
thx